engine/src/test/java/io/camunda/zeebe/engine/processing/incident/MultiInstanceIncidentTest.java

```diff
@@ -470,7 +469,7 @@ public void shouldCreateIncidentWhenInputCollectionModifiedConcurrently() {
             .processInstance(ProcessInstanceIntent.ELEMENT_COMPLETED, serviceTask.getValue()),
         RecordToWrite.event()
             .causedBy(2)
-            .key(nextKey)
+            .key(-1L)
```

Comment:
:thought_balloon: I'm totally unsure about this value, isn't `-1` also sometimes a special one? For what is the key used at the sequence flow? Do we have any element instance here :thinking:

Maybe @korthout or @saig0 can help here since I feel a bit uncomfortable about this.

Reply:
I needed some key and since it didn't matter I figured it'd be better to go with the obvious made up arbitrary value. I could add any other number here but I think none will make sense.